Buildings/Controls/OBC/OutdoorLights/DaylightControlled.mo

```diff
@@ -0,0 +1,99 @@
+within Buildings.Controls.OBC.OutdoorLights;
+block DaylightControlled "Controlling the outdoor lighting based on daylight time"
+  parameter Modelica.SIunits.Angle lat(displayUnit="deg") "Latitude";
+  parameter Modelica.SIunits.Angle lon(displayUnit="deg") "Longitude";
+  parameter Modelica.SIunits.Time timZon(displayUnit="h") "Time zone of location";
+
+  CDL.Interfaces.RealOutput y "Output on/off control signal"
+   annotation (Placement(transformation(extent={{100,-10},{120,10}})));
+
+protected
+  CDL.Utilities.SunRiseSet sunRiseSet(
+  final lat=lat,
+  final lon=lon,
+  final timZon=timZon)
+   annotation (Placement(transformation(extent={{-20,-10},{0,10}})));
+
+equation
+  y = if sunRiseSet.sunUp then 0 else 1;
```

Comment:
While this is valid Modelica, it is not valid in OpenBuildingControl. See http://obc.lbl.gov/specification/cdl.html#equations

The CDL translator modelica-json won't translate such assignments.

Reply:
CDL blocks are used for the implementation. Please review.